### PR TITLE
ENCD-4330 Stop using _all for indexing

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: src/encoded/viewconfigs/matrix.py [hunk 1]
```diff
@@ -61,7 +62,7 @@ def _construct_query(self):
             del query['query']['query_string']
         else:
             query['query']['query_string']['fields'].extend(
-                ['_all', '*.uuid', '*.md5sum', '*.submitted_file_name']
+                [RESOURCES_INDEX, '*.uuid', '*.md5sum', '*.submitted_file_name']
```
Comment (keenangraham) [inline src/encoded/viewconfigs/matrix.py]: These should be field names not indices names?

**Comment by keenangraham** [comment]
I think the confusing thing here is that `_all` is being used differently depending on if it is Python ES API or ES mapping? Right @hitz?

File: src/encoded/viewconfigs/matrix.py (hunk 1 above)
Comment (hitz) [reply src/encoded/viewconfigs/matrix.py]: no it's chr indexes that has to be skipped.

Comment (keenangraham) [reply src/encoded/viewconfigs/matrix.py]: Yeah but these are `fields` not indices.

Comment (caseylitton) [reply src/encoded/viewconfigs/matrix.py]: reverting this change.

**Comment by caseylitton** [comment]
Encoded does not need any changes associated with SNO-68.  Keeping branch to set snovault change.

**Pull request closed without merging**